src/SprykerEco/Zed/Payone/Business/PayoneBusinessFactory.php

```diff
@@ -357,6 +360,16 @@ protected function createGenericPayment(Store $store)
         return $genericPayment;
     }
 
+    /**
+     * @param \Spryker\Shared\Kernel\Store $store
+     *
+     * @return \SprykerEco\Zed\Payone\Business\Payment\MethodMapper\KlarnaMapperInterface
+     */
+    public function createKlarna(Store $store): KlarnaMapperInterface
+    {
+        return new Klarna($store, $this->getProvidedDependency(PayoneDependencyProvider::SERVICE_REQUEST_STACK));
```

Comment:
* Extract the getter of the `SERVICE_REQUEST_STACK` to the `getRequestStack()` method.

Reply:
++

Reply:
++